### ISSUE TITLE
Improve "Server Spoof" (still detected by wraith AntiCheat as non vanilla)

### ISSUE THREAD

**Issue by cizz0r** [MeteorDevelopment/meteor-client#3296]
### Describe the feature

![grafik](https://user-images.githubusercontent.com/111606933/216790223-8897baa3-40b1-4650-be1b-6b766c8301d3.png)

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by I-Got-You-Dead** [comment]
Send ur config

**Comment by PalmTamino** [comment]
> Send ur config

I can do it tomorrow. I think it detects it by making the client execute a meteor command? And if it doesn’t return something, they know it.

**Comment by ThebestkillerTBK** [comment]
add fabric to blocked channels

**Comment by ThebestkillerTBK** [comment]
#2404

**Comment by cizz0r** [comment]
> add fabric to blocked channels
Nope, the plugin has more ways of detecting it. I already hid all channels as seen in the screenshot.

**Comment by cizz0r** [comment]
> Send ur config

How can I do this?
(The .nbt file??)

**Comment by ThebestkillerTBK** [comment]
> > add fabric to blocked channels
> > Nope, the plugin has more ways of detecting it. I already hid all channels as seen in the screenshot.

What channels are blocked? Send screenshot
